docs/src/main/paradox/release-notes/10.2.x.md

```diff
@@ -40,19 +45,42 @@ we release 10.2.0 proper.
 * Better support for the new Actors API [#3036](https://github.com/akka/akka-http/pull/3036)
 * Parse empty query `?` to `Query.empty` [#3042](https://github.com/akka/akka-http/pull/3042)
 * Make sure to cancel response entity on failure [#3046](https://github.com/akka/akka-http/pull/3046)
-* add `akka.http.server.remote-address-attribute` [#2924](https://github.com/akka/akka-http/pull/2924)
+* Add `akka.http.server.remote-address-attribute` [#2924](https://github.com/akka/akka-http/pull/2924)
 * Make `transparent-head-requests` opt-in [#3063](https://github.com/akka/akka-http/pull/3063)
+* Allow multiple Host headers in responses [#3158](https://github.com/akka/akka-http/pull/3158)
 * Allow customizing parsing errors [#3049](https://github.com/akka/akka-http/pull/3049)
+* Don't extend from Char => Boolean for CharPredicate [#3107](https://github.com/akka/akka-http/pull/3107)
```

Comment:
Ultimately, we should point out the performance improvements as such (and not as its technical details) but can wait for the final announcement.